python/ray/tests/test_metrics_agent.py

```diff
@@ -277,54 +105,55 @@ def fetch_prometheus(prom_addresses):
             if address not in components_dict:
                 components_dict[address] = set()
             try:
-                response = requests.get(
-                    "http://localhost:{}".format(metrics_export_port))
+                response = requests.get(f"http://{address}/metrics")
```

Comment:
@rkooo567 this was the bug causing the test to be flaky/fail earlier. We were originally just pinging one exporter but we should ping them all.